src/main/target/DIAT_MAMBAF405_2022B/target.mk

```diff
@@ -0,0 +1,14 @@
+F405_TARGETS   += $(TARGET)
+FEATURES       += VCP ONBOARDFLASH
+
+TARGET_SRC = \
+drivers/accgyro/accgyro_mpu.c \
+drivers/accgyro/accgyro_mpu6500.c \
+drivers/accgyro/accgyro_spi_mpu6000.c \
+drivers/accgyro/accgyro_spi_mpu6500.c \
+drivers/accgyro/accgyro_spi_icm426xx.c \
+drivers/max7456.c
+
```

Comment:
Might want to add other baro and mag defines as this board does have I2C ports and external gps and baro could be added. Also want to include light strip define as this board does have led strip.
						drivers/barometer/barometer_bmp085.c \
						drivers/barometer/barometer_bmp280.c \
						drivers/barometer/barometer_ms5611.c \
						drivers/compass/compass_hmc5883l.c \
						drivers/compass/compass_qmc5883l.c \
						drivers/light_ws2811strip.c\